src/content/en/updates/2016/09/project_path/web/_project.yaml-book_path/web/updates/_book.yaml-description-BroadcastChannel-API-can-be-used-for-simple-pub/re-rastering-composite.md

```diff
@@ -0,0 +1,91 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: 
+
+{# wf_updated_on: 2016-09-22 #}
+{# wf_published_on: 2016-09-22 #}
+{# wf_tags: css,webanimations #}
+
+# Re-rastering Composited Layers on Scale Change {: .page-title }
+
+{% include "_shared/contributors/chrishtr.html" %}
+
+## TL;DR
```

Comment:
Hey @jpmedley - `## TL;DR`'s should be `### TL;DR {: .hide-from-toc }` Can you make that change please.